app/views/wrangling_guidelines/show.html.erb

```diff
@@ -8,7 +8,7 @@
 
 <!--main content-->
 <div class="admin" role="article">
-  <% if logged_in_as_admin? %>
+  <% if policy(WranglingGuideline).new? %>
```

Comment:
Since we're deciding whether to display the edit link, let's use `policy(WranglingGuideline).edit?` instead.